### ISSUE TITLE
Touch Effects: fix labels for buttons in settings

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#234]
They're buttons so technically don't need labels, but some of them are Off/On so don't make sense out of context. We need to think how to do it better. They kind of feel like they should be radios.
